sass/index.scss

```diff
@@ -435,6 +436,119 @@ video::-webkit-media-controls {
     }
 }
 
+// Prizes
+#prizes {
+    $tape-offset: 35px;  // how far into the card is the tape placed?
+    
+    @mixin tape-transform($rotation) {
+        transform: translate(-50%, -50%) rotate($rotation);
+        // this makes it so that we're positioning the tape by its center instead of its top-left, pre-rotation
+        // corner, which is significantly easier
+    }
+
+    #prizes-ricky {
+        // placeholder
+        width: 85%;
+        max-width: 980px;
+        height: auto;
+        background-color: blue;
+        margin: 0 auto;
+    }
+
+    #prizes-card {
+        background-color: white;
+        border: 1px solid $light-blue;
+        border-radius: 6px;
+        margin-top: -195px;  // need to overlap prizes ricky
+        padding: 75px 95px;
+        position: relative;
+        width: 100%;
+
+        .tape {
+            background-color: $beige;
+            height: 40px;
+            opacity: 25%;
+            position: absolute;
+            width: 165px;
+
+            &.top-left {
+                @include tape-transform(-45deg);
+                left: $tape-offset;
+                top: $tape-offset;
+            }
+
+            &.top-right {
+                @include tape-transform(45deg);
+                right: $tape-offset - 165px;
+                top: $tape-offset;
+            }
+
+            &.bottom-left {
+                @include tape-transform(45deg);
+                left: $tape-offset;
+                bottom: $tape-offset - 40px;
+            }
+
+            &.bottom-right {
+                @include tape-transform(-45deg);
+                right: $tape-offset - 165px;
+                bottom: $tape-offset - 40px;
+            }
+        }
+
+        & > .content {
```

Comment:
shouldnt `.content` alone work?

Reply:
I think she made another div with the class content inside the inner-prizes-card and doesn't want to select that